dnsserv_test.go

```diff
@@ -1,40 +0,0 @@
-package main
-
-import (
-	"bytes"
-	. "gopkg.in/check.v1"
-	"net"
-	"testing"
-)
-
-func Test(t *testing.T) { TestingT(t) }
-
-type DNSNameserverSuite struct{}
-
-var _ = Suite(&DNSNameserverSuite{})
-
-func (s *DNSNameserverSuite) TestUDPConnection(c *C) {
```

Comment:
The way this test is constructed can definitely be changed, but we should test logic around connecting, listening, and responding to errors